### PR TITLE
[SPARK-10254][ML]Removes Guava dependencies in spark.ml.feature JavaTests

### PR DESCRIPTION
**Pull request by feynmanliang**
- Replaces `com.google.common` dependencies with `java.util.Arrays`
- Small clean up in `JavaNormalizerSuite`

### CODE REVIEW

File: mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java [hunk 1]
```diff
@@ -17,7 +17,8 @@
 
 package org.apache.spark.ml.feature;
 
-import com.google.common.collect.Lists;
+import java.util.Arrays;
```
Comment (mengxr) [inline mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java]: organize imports

**Comment by mengxr** [comment]
LGTM except that imports needs to be organized.

File: mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java (hunk 1 above)
Comment (feynmanliang) [inline mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java]: What is the correct import order for java? I followed the scala import guidelines here

File: mllib/src/test/java/org/apache/spark/ml/feature/JavaTokenizerSuite.java [hunk 2]
```diff
@@ -29,6 +29,8 @@
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SQLContext;
 
+import java.util.Arrays;
```
Comment (mengxr) [inline mllib/src/test/java/org/apache/spark/ml/feature/JavaTokenizerSuite.java]: Sorry, I meant this one.

File: mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java (hunk 1 above)
Comment (mengxr) [inline mllib/src/test/java/org/apache/spark/ml/feature/JavaBucketizerSuite.java]: Sorry, wrong line. See my comment below.

File: mllib/src/test/java/org/apache/spark/ml/feature/JavaTokenizerSuite.java (hunk 2 above)
Comment (feynmanliang) [inline mllib/src/test/java/org/apache/spark/ml/feature/JavaTokenizerSuite.java]: Thanks, fixed.

**Pull request closed without merging**